src/extension-runners/index.js

```diff
@@ -31,6 +31,7 @@ export type MultiExtensionRunnerParams = {|
 
 // Export everything exported by the firefox-desktop runner.
 export * from './firefox-desktop';
+export * from './firefox-android';
```

Comment:
`export * from '...'` only exports what the module has exported explicitly and not what they have imported inside the module it.

Nevertheless I'm totally ok to change it if we don't like it because it is not explicit enough.